### DISCUSSION TITLE
CMB: Troubling echoes of the Big Bang | Matt O'Dowd, Marika Taylor, and Martín López-Corredoira

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#287] [General]
Thirteen minutes of the debate, not behind a paywall https://youtu.be/Qn22mPuct9w

Paywalled version https://iai.tv/video/troubling-echoes-of-the-big-bang

Great to see ACG members Bjørn Ekeberg and Martín López-Corredoira together!

**Comment by budrap00** [comment]
The funniest comment was O'Dowd saying he found the descriptive term _ad hoc_ to be judgmental.

**Reply by RedshiftDrift** [reply]
... which makes me suspect O'Dowd doesn't know what *ad hoc* means.

**Comment by RedshiftDrift** [comment]
Is The Big Bang Theory Controlling How We See CMBR Data?
https://youtu.be/Ku7mjp25Uq0

**Reply by budrap00** [reply]
I have to admit that is the most impressively open-minded discussion of the CMB by a mainstream scientist I have yet encountered. It was especially gratifying to see him point out the circular reasoning that lies at the heart of how cosmological data is typically interpreted. He also managed to discuss plasma issues without referring to "electric universe nuts".

The end of the Expanding Universe as the only allowed cosmological framework for interpreting cosmological data is probably not quite at hand, but there does seem to be a growing body of dissidents within the cosmological community. Dissidents meaning cosmologists open to the _possibility_ that the EU framework _might_ be wrong. That alone would be a major advance in turning modern cosmology from a belief system into a science.
